src/modules/removeItem.js

```diff
@@ -0,0 +1,10 @@
+import { toDoList } from './addItem.js';
+
+export default window.removeItem = (removeItemId) => {
+  const remItemArr = toDoList.filter((eachItem) => eachItem.index !== removeItemId);
+  for (let i = 0; i < remItemArr.length; i += 1) {
+    remItemArr[i].index = i + 1;
+  }
```

Comment:
In removeItem.js file, instead old for you can use forEach loop, since it is alittle hard to handle the code with old for. :relaxed: